### PR TITLE
[15.0][MIG] pos_supplierinfo_barcode

### PR DESCRIPTION
**Pull request by Rad0van**
Straightforward migration to 15.0

### CODE REVIEW

File: pos_supplierinfo_barcode/models/product.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import json
+
+from odoo import fields, models
+
+
+class ProductTemplate(models.Model):
+    _inherit = "product.template"
+
+    # technical field used in POS frontend
+    supplier_barcode_json = fields.Char(
```
Comment (ivantodorovich) [inline pos_supplierinfo_barcode/models/product.py:10]: I recommend renaming this field to `pos_supplier_barcode_json` and to move this comment to the field's `help` attribute.


Not blocking though.

**Review by ivantodorovich — APPROVED** [review]

**Comment by ivantodorovich** [comment]
Could you check pre-commit? seems off

**Review by GabbasovDinar — APPROVED** [review]
Code LGTM!

**Pull request merged**